_includes/yaml_download.md

```diff
@@ -4,15 +4,16 @@ Include:
 - filename
```

Comment:
we should remove `- data` from the list of includes, since now that's an `assign`

Reply:
Good catch! I'll remove.